visualization/CHANGELOG.md

```diff
@@ -29,7 +31,6 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 ### Fixed 🐞
 
 - Revert [#3655](https://github.com/MaibornWolff/codecharta/pull/3665) as we implement new navigation methods
-- Camera perspective is correctly adopted from the custom configuration[#3194](https://github.com/MaibornWolff/codecharta/pull/3194)
```

Comment:
Why do we need to change the changelog of previous versions?

Reply:
Actually, that's a recent bug fix, it has nothing to do with the previous version. It was there by mistake, and I corrected it